shell/platform/android/io/flutter/embedding/engine/android/FlutterView.java

```diff
@@ -180,6 +180,9 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
    */
   @Override
   public final WindowInsets onApplyWindowInsets(WindowInsets insets) {
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.KITKAT_WATCH) {
```

Comment:
Can you add a comment about why returning null is the correct behavior for earlier SDKs?

Reply:
The Javadoc on this method mentions that this isn't called on API < 20. Do you think that's sufficient, or do you want more detail there saying that if it is deliberately called on those SDKs it'll return null as a non-op?

Reply:
Perhaps add a `@TargetApi` or `@RequiresApi` as appropriate here?